src/app/host/[roomId]/components/RegisterFormProvider.tsx

```
@@ -0,0 +1,183 @@
+        return;
+      case 'structure':
+        const structure = formData.get('structure') as string;
+        updateData = { structure };
+        break;
+      case 'privacy':
+        const privacy = formData.get('privacy') as string;
+        updateData = { roomType: privacy };
+        break;
+      case 'location':
+        const location = formData.get('location') as string;
+        const latitude = Number(formData.get('latitude'));
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)